gateway/src/api/latest.rs

```diff
@@ -167,6 +177,7 @@ pub fn make_api(service: Arc<GatewayService>, sender: Sender<BoxedTask>) -> Rout
         )
         .route("/users/:account_name", get(get_user).post(post_user))
         .route("/projects/:project/*any", any(route_project))
+        .route("/admin/revive", get(revive_projects))
```

Comment:
```suggestion
        .route("/admin/revive", post(revive_projects))
```
seems more idiomatic? Under the idea that you'd have `/admin/revive/0`, `/admin/revive/1` etc to describe operations. Obviously we don't have that yet, but I think this would be more intuitive than using `get` for starting side-effects.

Reply:
Agreed. I wasn't sure about this one since there's no body, but I looked around now and there seems to be some consensus on empty POST being okay for cases like this.